frontend/src/components/Board/Board.styled.jsx

```diff
@@ -10,17 +9,19 @@ export const BoardInfoContainer = styled.div`
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
   background: #e3e3e3;
   color: #2c2c2c;
-`
+`;
 
 export const BoardInfo = styled.div`
   position: relative;
   min-width: 100%;
   display: flex;
   align-items: center;
   overflow-wrap: break-word;
```

Comment:
```suggestion
  display: flex;
  align-items: center;
  overflow-wrap: break-word;
  min-width: 100%;

```
Tak żeby trzymać się konwencji :)